nav2_controller/src/nav2_controller.cpp

```diff
@@ -81,32 +90,31 @@ DwbController::on_configure(const rclcpp_lifecycle::State & state)
 
   // Create the action server that we implement with our followPath method
   action_server_ = std::make_unique<ActionServer>(rclcpp_node_, "FollowPath",
-      std::bind(&DwbController::followPath, this));
+      std::bind(&nav2_controller::followPath, this));
 
   return nav2_util::CallbackReturn::SUCCESS;
 }
 
 nav2_util::CallbackReturn
-DwbController::on_activate(const rclcpp_lifecycle::State & state)
+nav2_controller::on_activate(const rclcpp_lifecycle::State & state)
 {
   RCLCPP_INFO(get_logger(), "Activating");
 
-  planner_->on_activate(state);
+  local_planner_->on_activate(state);
```

Comment:
so we're calling the package `nav2_controller` and we're changing alot of the naming to `controller` (which I'm ok with) but then we should kill all references to `local_planner` so we're not mixing terminology. `controller_` would be a good substitute

Reply:
@SteveMacenski  The ```local_planner``` here is referring to the plugin (dwb), so do we want to change dwb local ```planner```,  references to the controller? Or do you just want it to be changed here?

Reply:
>  so do we want to change dwb local planner, references to the controller

Potentially. @crdelsey am I being too knit picky on this one? Maybe its clear to everyone but I think I've just been thinking about this area for too long to have a objective understanding if there's even a confusion needing to be resolved